src/cell.ts

```diff
@@ -1,34 +1,27 @@
-import { AdminApi } from '@holochain/conductor-api'
+import { AdminApi, AppApi, CellId } from '@holochain/conductor-api'
+import { fakeCapSecret } from './common'
 
 type CallZomeFunc = (zome: string, fn: string, params: any) => Promise<any>
 
 type CellConstructorParams = {
-  nick: string,
-  dnaAddress: string,
-  agentAddress: string,
+  cellId: CellId,
   adminClient: AdminApi,
-  callZome: CallZomeFunc,
+  appClient: AppApi,
 }
 
 /**
- * Handy reference to an cell within a Conductor.
- * Rather than using conductor.call('appId', 'cellNick', 'zomeName', 'funcName', params), you can:
- * `conductor.cells[cellNick].call('zomeName', 'funcName', params)`
+ * Make zome calls from Cells.
+ * `cell.call('zomeName', 'funcName', params)`
  */
 export class Cell {
-
-  nick: string
+  cellId: CellId
   admin: AdminApi
-  _callZome: CallZomeFunc
-  agentAddress: string
-  dnaAddress: string
+  app: AppApi
 
   constructor(o: CellConstructorParams) {
-    this.nick = o.nick
+    this.cellId = o.cellId
+    this.app = o.appClient
```

Comment:
I think we should call these `appInterface` and `adminInterface` because as they are they are confusing.